src/entries/popup/hooks/useInfiniteTransactionList.ts

```diff
@@ -17,12 +18,14 @@ import { useTestnetModeStore } from '~/core/state/currentSettings/testnetMode';
 import { useCustomNetworkTransactionsStore } from '~/core/state/transactions/customNetworkTransactions';
 import { RainbowTransaction } from '~/core/types/transactions';
 import { useSupportedChains } from '~/core/utils/chains';
+import { RainbowError, logger } from '~/logger';
 
-import useComponentWillUnmount from './useComponentWillUnmount';
 import { useKeyboardShortcut } from './useKeyboardShortcut';
 import { useUserChains } from './useUserChains';
 
-const PAGES_TO_CACHE_LIMIT = 2;
+// We have page sizes of 100 since goldsky migration, so we only need to cache the first page on navigate away (these pages will be revalidated on return to the page, so we should keep as little pages as possible in cache)
+const PAGES_TO_CACHE_LIMIT = 1;
+const FIRST_PAGE_REFETCH_INTERVAL = 60000; // 1 minute
```

Comment:
Noticed that all mounts (refresh, switch tabs, etc.) escape this interval and trigger a refresh for the first page right away. Seems unnecessary because we wouldn't expect transactions to regularly change unless the chain list changes OR an interaction occurred, and we already force a refresh there. I suppose tokens makes more sense for that behavior, but we do have the new lighter live balance/pricing endpoint for that which give us a delta update instead of the full asset list. I also noticed that it seems like we're not passing the `timestamp` to `GetAssetsUpdates` so we are getting the full payload each time